lib/ncr/mailboxes.rb

```diff
@@ -0,0 +1,29 @@
+module Ncr
+  module Mailboxes
+    def self.ba61_tier1_budget
+      self.approver_with_role("BA61_tier1_budget_approver")
+    end
+
+    def self.ba61_tier2_budget
+      self.approver_with_role("BA61_tier2_budget_approver")
+    end
+
+    def self.ba80_budget
+      self.approver_with_role("BA80_budget_approver")
+    end
+
+    def self.ool_ba80_budget
+      self.approver_with_role("OOL_BA80_budget_approver")
+    end
+
+    protected
+
+    def self.approver_with_role(role_name)
```

Comment:
protected class methods are tricky: http://blog.jayfields.com/2006/11/ruby-protected-class-methods.html

what about making the methods in here instance methods instead and then we can use `private` for `approver_with_role` ?